plugin/trino-blackhole/src/test/java/io/trino/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -117,18 +116,14 @@ public void notAllPropertiesSetForDataGeneration()
                 .setSchema("default")
                 .build();
 
-        try {
-            assertThatQueryReturnsValue(
-                    format("CREATE TABLE nation WITH ( %s = 3, %s = 1 ) as SELECT * FROM tpch.tiny.nation",
-                            ROWS_PER_PAGE_PROPERTY,
-                            SPLIT_COUNT_PROPERTY),
-                    25L,
-                    session);
-            fail("Expected exception to be thrown here!");
-        }
-        catch (RuntimeException ex) {
-            // expected exception
-        }
+        assertThatThrownBy(() -> assertThatQueryReturnsValue(
```

Comment:
replace `assertThatQueryReturnsValue` with eg `queryRunner.execute` -- the intent here is _not_ to run an assertion on the results.